### DISCUSSION TITLE
Display a count of unsynced events on the App

### DISCUSSION THREAD

**Discussion by tychus6** [opensrp/opensrp-client-path-zeir#150] [General]
We received a request from ZEIR team, they would like the App to display a count of unsynced events .
The count could  either be one way or two way, for one  way ,the app should display a count of the  events on the  App that have not been synced with server, while for two way  it should display a count of unsynced events in both the App and Server for that particular location. 

Constraints.

- Most facilities have poor internet connectivity, therefore the proposed implementation should  to take this  into consideration .
What are your thoughts on this?
-cc @ellykits @ndegwamartin @AnnieMungai

**Comment by AnnieMungai** [comment]
@tychus6 @ndegwamartin my concern would be the LOE that this demands. Does it even fall under the OpenSRP roadmap or it is something FHIR could potentially take care of? 

But yes, we might need a way to show the client whether they have synced all the records that are in the device.

**Reply by ndegwamartin** [reply]
@AnnieMungai @ellykits I think this is already implemented for the Reveal project. 

For LOE we probably just need to confirm how much of the generic code has been abstracted into the core library and how much is needed for the implementing app. Either way I don't expect it will be much of a heavy lift, but best to confirm this.

**Reply by AnnieMungai** [reply]
@tychus6 could possibly help us with this? 
@ndegwamartin thanks! I didn't know that exists.

**Reply by tychus6** [reply]
@AnnieMungai Sure ,i'l do a follow up  with the reveal team, also as  per our meeting, the most viable option is the one way sync since  it's not dependant on internet connectivity cc @ndegwamartin @ellykits

**Reply by AnnieMungai** [reply]
@tychus6 what is the LOE required for that? I would need to see that before we implement it. Thanks!
